### ISSUE TITLE
Column width wrong when values contain new line characters

### ISSUE THREAD

**Issue by GitMyCode** [someatoms/jsPDF-AutoTable#29]
Hi!
I wanted a column 'textarea' with breaklines. but the width of the column was computed on the all the char like it was on one line.

![badwidth](https://cloud.githubusercontent.com/assets/3288428/7556236/c36e2a30-f736-11e4-9412-5e70a41e5875.PNG)

i had to quick fix it with this code
```javascript
 function getStringWidth(txt) {
        var isBold = doc.internal.getFont().fontStyle === 'bold';

        var textWithBreakLine = txt.split("\n");
        if (textWithBreakLine.length > 1) {
            var widest = 0;
            textWithBreakLine.forEach(function(line) {
                var lineWidth = doc.getStringUnitWidth(line) * doc.internal.getFontSize() + (isBold ? 5 : 0);
                widest = (lineWidth > widest) ? lineWidth : widest;
            });
            return widest;
        } else {
            return doc.getStringUnitWidth(txt) * doc.internal.getFontSize() + (isBold ? 5 : 0);
        }
    }
```

The result is this

![whatiwant](https://cloud.githubusercontent.com/assets/3288428/7556244/0304570a-f737-11e4-8108-8361f937055c.PNG)

I just wanted to mention it and i suppose it wouldn't be a problem with the feature of a custom column width  #20

**Comment by someatoms** [comment]
I will look into this and probably do something similar to your fix. I realized there are other issues with putting line breaks in the value so I will try to address those at the same time.  I implemented custom column width in version `1.3` so for now (as you mentioned) that can be used. Thanks!

**Comment by someatoms** [comment]
This should now be fixed in version 2.0.
